concent_api/core/models.py

```diff
@@ -44,11 +61,11 @@ class Meta:
         verbose_name_plural = ('ReceiveOutOfBand statuses')
 
 
-class MessageAuth(models.Model):
+class MessageAuth(Model):
```

Comment:
This should be a fixup to the commit that adds the model. It does not belong in this commit.